code/game/machinery/computer/prisoner.dm

```diff
@@ -17,6 +17,14 @@
 
 	light_color = LIGHT_COLOR_DARKRED
 
+/obj/machinery/computer/prisoner/New()
+	prisoncomputer_list += src
+	return ..()
```

Comment:
I don't think you need to have this return ..(), a simple ..() will probably suffice since I believe New() doesn't have return values like Destroy() does.

Someone may correct me if I'm wrong, but I can't think of anything that has a return value in their New() proc